create-rector.yaml

```diff
@@ -0,0 +1,33 @@
+parameters:
```

Comment:
This file is ignored by `.gitignore`, it should not be part of resposity. It's only for local development.